twitter/View Controllers/TimelineViewController.m

```diff
@@ -10,31 +10,51 @@
 #import "APIManager.h"
 #import "AppDelegate.h"
 #import "LoginViewController.h"
+#import "TweetCell.h"
+#import "Tweet.h"
+#import "UIImageView+AFNetworking.h"
 
-@interface TimelineViewController ()
+@interface TimelineViewController () <UITableViewDelegate,UITableViewDataSource>
```

Comment:
I'm glad to see that you're using `UITableViewDelegate` here instead of segues! Awesome! :)